docker-images/build.sh

```diff
@@ -119,6 +119,17 @@ function fetch_and_unpack_kafka_binaries {
         # Store the folder address for use in the image build 
         version_dist_dirs["$kafka_version"]="./tmp/$kafka_version"
 
+        # create a file listing all the jars with colliding class files in the Kafka dist
+        # (on the assumption that this is OK). This is used after building the images to detect any collisions
```

Comment:
This comment confused me a bit - I assume that it is the file which is used after building the  images, or?

```suggestion
        # (on the assumption that this is OK). This file will be used after building the images to detect any collisions
```